pairing/pairing.py

```diff
@@ -100,6 +100,25 @@ def _find_intersection(a, b):
     intersection = np.maximum(a, b)
     return intersection
 
+def _check_validity(c_I):
+    """
+    Check validity of indirect connectivity matrix
+
+    Parameters
+    ----------
+    c_I : np.ndarray
+    indirect connectivity matrix to test
+
+    Returns
+    -------
+    Boolean 'True' or 'False'
+    """
+
+    test_indirect = generate_indirect_connectivity(c_I)
+    if (test_indirect == c_I).all():
+        return True
+    else:
+        return False
```

Comment:
What about `return (test_indirect == c_I).all()`?

Reply:
Good point, that works just as well.